configargparse.py

```diff
@@ -836,12 +836,15 @@ def error_method(self, message):
 
             if not user_config_file:
                 continue
-            # validate the user-provided config file path
+
+            # open user-provided config file
             user_config_file = os.path.expanduser(user_config_file)
-            if not os.path.isfile(user_config_file):
+            try:
+                stream = self._config_file_open_func(user_config_file)
+            except OSError:
```

Comment:
Are you sure OSError is the right error type to catch here? what is a custom _config_file_open_func throws a different type of error?
I think `except Exception as e` might be more general.
Also, the call may fail for many different reasons besides  "file not found" (eg. permissions denied). Please change the error message to `"Unable to open config file: %s. Error: %s" % ( user_config_file, e)`